src/TorchSharp/Utils/tensorboard/protobuf/Graph.cs

```diff
@@ -0,0 +1,310 @@
+// <auto-generated>
+//     Generated by the protocol buffer compiler.  DO NOT EDIT!
+//     source: tensorboard/compat/proto/graph.proto
+// </auto-generated>
+#pragma warning disable 1591, 0612, 3021
+#region Designer generated code
```

Comment:
IMHO, non-torch specific parts of the TensorBoard code should be built as a separate NuGet package.

Reply:
You are definitely right on principle. Right now, with the minimal feature set, I think I'll keep it in TorchSharp, but we can definitely pull it out later. It might be useful for ML.NET, too.